### PR TITLE
Update Potential_IIS_CodeInject.yaml

### PR DESCRIPTION
**Pull request by 4R9UN**
using "ipv4_is_private()" operator to identify the private IP, reduced noise from loopback IP

   Required items, please complete
   
   Change(s):
    Update Potential_IIS_CodeInject.yaml
     using "ipv4_is_private()" operator to identify the private IP, exclude loopback IP . 
  
   Reason for Change(s):
   - generated FP results 

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

**Review by 4R9UN — COMMENTED** [review]
done changes related to YAML

**Review by aprakash13 — COMMENTED** [review]
Thanks @4R9UN . The changes look great. Please remove the extra line in the end and this should be ready to be merged.

**Review by 4R9UN — COMMENTED** [review]
@aprakash13, removed the blank lines, ready for the merge.

**Comment by shainw** [comment]
@4R9UN - Please add the entity mapping fields at the end for the query, same as we do for detections.  We are doing this to support future functionality.

File: Hunting Queries/W3CIISLog/Potential_IIS_CodeInject.yaml [hunk 1]
```diff
@@ -18,16 +18,15 @@ query: |
...
   // Exclude common csMethods, add/modify this list as needed for your environment
   let csMethodExclude = dynamic(['GET', 'DEBUG', 'DELETE', 'LOCK', 'MKCOL', 'MOVE', 'PATCH', 'POST', 'PROPPATCH', 
   'PUT', 'SEARCH', 'TRACE', 'TRACK', 'UNLOCK', 'OPTIONS', 'HEAD', 'RPC_IN_DATA', 'RPC_OUT_DATA', 'PROPFIND','BITS_POST','CCM_POST']);
   // Include in the list expected IPs where remote methods such as vuln scanning may be expected for your environment
   let expectedIPs = dynamic(['X.X.X.X', 'Y.Y.Y.Y']);
   let codeInjectionAttempts = W3CIISLog
-  | extend cIPType = iff(cIP matches regex PrivateIPregex,"private" ,"public" )
-  | where cIPType =="public"
+  // Exclude private ip ranges from cIP list
+  |where cIP != "::1"
```
Comment (shainw) [inline Hunting Queries/W3CIISLog/Potential_IIS_CodeInject.yaml:28]: Please add a space in front of the where clause here to align with standards.

**Review by shainw — CHANGES_REQUESTED** [review]

**Comment by aprakash13** [comment]
> @4R9UN - Please add the entity mapping fields at the end for the query, same as we do for detections. We are doing this to support future functionality.

When you have some time @4R9UN please have a look at this suggestion. Thanks

**Comment by 4R9UN** [comment]
Hi @aprakash13 , Thanks for the reminder, I added Entity mapping, kindly review the query and let me know if anything is missing.

**Review by aprakash13 — APPROVED** [review]
Thanks @4R9UN

**Pull request merged**